### PR TITLE
disabling coverages in test code

### PR DESCRIPTION
**Pull request by lucasborin**

### CODE REVIEW

File: src/checks/y_check_branch_coverage.clas.abap [hunk 1]
```diff
@@ -18,7 +18,7 @@ CLASS y_check_branch_coverage IMPLEMENTATION.
     settings-threshold = 100.
     settings-is_threshold_reversed = abap_true.
     settings-disable_on_prodcode_selection = abap_true.
-    settings-disable_on_testcode_selection = abap_true.
+    settings-disable_on_testcode_selection = abap_false.
```
Comment (eugen-guenther-sap) [inline src/checks/y_check_branch_coverage.clas.abap:21]: you just enabled the testcode selection and not disabled anything.

you need: 
settings-apply_on_test_code = abap_false.

File: src/checks/y_check_procedure_coverage.clas.abap [hunk 2]
```diff
@@ -18,7 +18,7 @@ CLASS y_check_procedure_coverage IMPLEMENTATION.
     settings-threshold = 100.
     settings-is_threshold_reversed = abap_true.
     settings-disable_on_prodcode_selection = abap_true.
-    settings-disable_on_testcode_selection = abap_true.
+    settings-disable_on_testcode_selection = abap_false.
```
Comment (eugen-guenther-sap) [inline src/checks/y_check_procedure_coverage.clas.abap:21]: same here

File: src/checks/y_check_statement_coverage.clas.abap [hunk 3]
```diff
@@ -18,7 +18,7 @@ CLASS y_check_statement_coverage IMPLEMENTATION.
     settings-threshold = 100.
     settings-is_threshold_reversed = abap_true.
     settings-disable_on_prodcode_selection = abap_true.
-    settings-disable_on_testcode_selection = abap_true.
+    settings-disable_on_testcode_selection = abap_false.
```
Comment (eugen-guenther-sap) [inline src/checks/y_check_statement_coverage.clas.abap:21]: and here

**Review by eugen-guenther-sap — CHANGES_REQUESTED** [review]
see comments

File: src/checks/y_check_branch_coverage.clas.abap (hunk 1 above)
Comment (lucasborin) [reply src/checks/y_check_branch_coverage.clas.abap]: oh, well observed. Thanks for it!

**Comment by lucasborin** [comment]
@eugen-guenther-sap: Could you please check it again?

**Review by eugen-guenther-sap — APPROVED** [review]

**Pull request merged**